tests/plugins/test_resolve_composes.py

```diff
@@ -178,13 +175,25 @@ def mock_repo_config(tmpdir, data=None, signing_intent=None):
     tmpdir.join('container.yaml').write(data)
 
 
+def mock_container_set_config(tmpdir, data=None):
+    if data is None:
+        data = dedent("""\
+            compose:
```

Comment:
As noted in an earlier comment, this line should be removed.

Reply:
Fixed.